enderio-base/src/generated/.gitattributes

```diff
@@ -0,0 +1,3 @@
+# GSON uses LF only, this prevents annoyance on Windows where only EOLs differ
+*.json eol=lf
+*.zs eol=lf
```

Comment:
I don't think this is really needed for zs, as you aren't datagenning crafttweaker scripts.